Dockerfile

```diff
@@ -4,6 +4,16 @@ FROM hydroshare/hs_docker_base:0bc6b1d
 RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
     locale-gen
 
+RUN pip install django==4.2.*
+RUN pip install djangorestframework==3.14.*
+RUN pip install django-haystack==3.2.*
+RUN pip install drf-yasg==1.21.*
+RUN pip install django-robots==6.1
+RUN pip install django-autocomplete-light==3.9.*
```

Comment:
https://django-autocomplete-light.readthedocs.io/en/master/index.html

> Django 4.2 support starting DAL 3.10, support for earlier Django versions in DAL < 3.10

Reply:
There is now DAL 3.11